licenses/BSL.txt

```diff
@@ -7,16 +7,22 @@ Licensor:             Rocicorp, LLC
 Licensed Work:        replicache 6.2.0
                       The Licensed Work is (c) 2020 Rocicorp, LLC.
 
-Additional Use Grant: You may use the Licensed Work when your application uses
-                      the Licensed Work with a total of less than 100 client
-                      instances for any purpose.
+Additional Use Grant: You may make use of the Licensed Work, provided you
+                      do not use it in any of the following ways:
+
+                      * Create a synchronization engine. A "synchronization
+                     engine" is a commercial or non-commercial offering that
+                     synchronizes data between two or more computers.
+
+                     * Use it for commercial purposes with more than 500
+                     monthly active client instances.
```

Comment:
Do we want to switch the language to talk about "profile instances" or similar or should we just stick with "client" because it is pretty clear?

Reply:
Let's leave it as client for now as it is consistent with everything else.